src/main/java/com/collector/web/IndexServlet.java

```diff
@@ -0,0 +1,87 @@
+package com.collector.web;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+
+import Database.Interaction.Entities.Address;
+import Database.Support.CustomLogs;
+import Database.Support.DbConfig;
+import Database.Tables.T_Address;
+import com.collector.config.TemplateEngineUtil;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.context.WebContext;
+
+@WebServlet("/")
```

Comment:
dovod preco, sa vyuziva WebServlet() v takomto tvare a nie v takom v akom to bolo v povodom subore?